src/font.rs

```diff
@@ -35,9 +35,10 @@ impl FontRenderer {
 
         let metrics = self.font.metrics();
 
-        let em = size / 1.24;
-        let mut base_x = 0.;
+        let em = size; // width of character, larger values of optional factor (eg size * 1.1) increase spacing
```

Comment:
I'd prefer comment to go in the preceding line instead on the same line. This prevents very long lines from occurring.

Reply:
(Here and other places)